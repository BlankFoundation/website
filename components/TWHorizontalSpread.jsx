import {
  TWHorizontal
} from '.'

const TWHorizontalSpread = ({ className, children }) => {
  return (
    <TWHorizontal
      className={`${className} justify-between items-center`}
    >
      {children}
    </TWHorizontal>
  );
}

TWHorizontalSpread.defaultProps = {
  children: ['child1', 'child2', 'child3'].map(
    (child, index) => <div key={'child-' + index}>{child}</div>
  ),
  className: ""
}

export default TWHorizontalSpread;