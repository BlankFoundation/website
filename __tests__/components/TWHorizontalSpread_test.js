import React from 'react';
import TestRenderer from 'react-test-renderer';
import TWHorizontalSpread from '../../components/TWHorizontalSpread';

describe("TWHorizontalSpread", () => {
  it('renders correctly', () => {
    const rendered = TestRenderer.create(<TWHorizontalSpread />).toJSON();
    expect(rendered).toMatchSnapshot()
  });
});