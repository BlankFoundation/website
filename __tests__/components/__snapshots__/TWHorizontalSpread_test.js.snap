// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TWHorizontalSpread renders correctly 1`] = `
<div
  className=" justify-between flex flex-row"
>
  <div>
    child1
  </div>
  <div>
    child2
  </div>
  <div>
    child3
  </div>
</div>
`;
